bundles/net.tourbook/src/net/tourbook/nutrition/DialogCustomTourNutritionProduct.java

```diff
@@ -102,6 +102,8 @@ protected Control createDialogArea(final Composite parent) {
       }
       _isInUIInit = false;
 
+      updateUI();
+
```

Comment:
Should `updateUI()` not within `_isInUIInit = true/false`. I think `_isInUIInit` is only need when setting field contents to prevent that modify listener are doing their work

Reply:
The flag `_isIn...` is needed for some controls and events when the content of controls is modified programmatically and not by the user to prevent that modify events are not performed when they are fired. 

For some controls / events it is not needed but after 20 years of MT programming I cannot remember in my brain when it is needed and when not, so for me it is always a try and error.